rotkehlchen/chain/evm/accounting/structures.py

```diff
@@ -15,7 +16,7 @@ def __call__(
             self,
             pot: 'AccountingPot',
             event: 'EvmEvent',
-            other_events: Iterator['EvmEvent'],
+            other_events: "peekable['EvmEvent']",
```

Comment:
Adding this in `""` is only due to this limitation: https://github.com/python/mypy/issues/5264#issuecomment-399407428

Which is probably still an issue?

As mypy works fine, but when running the code it goes BOOM. With

```
rotkehlchen/chain/evm/accounting/structures.py:19: in EventsAccountantCallback                                                                                                                                                                 
    other_events: peekable['EvmEvent'],                                                                                                                                                                                                        
E   TypeError: 'type' object is not subscriptable 
```